pkg/version/version.go

```diff
@@ -22,8 +22,16 @@ import (
 	"runtime/debug"
 )
 
+// version to be set using ldflags:
+// -ldflags "-X sigs.k8s.io/controller-tools/pkg/version.version=1.0.0"
```

Comment:
```suggestion
// -ldflags "-X sigs.k8s.io/controller-tools/pkg/version.version=v1.0.0"
```
nit: Just to give a slightly more real example (CT used the v prefix in the past, I would continue to use it)